src/Features/LanguageServer/Protocol/Handler/RequestContext.cs

```diff
@@ -18,6 +18,10 @@ namespace Microsoft.CodeAnalysis.LanguageServer.Handler
     {
         private readonly Action<Solution>? _solutionUpdater;
 
+        /// <summary>
+        /// The document that the request is for, if applicable
```

Comment:
Worth mentioning with a cref that this is passed if the request overrode that other method, so you can better connect the two?